{"uid":"5b3926c3655b7d5b","name":"Тест на удаление несуществующего сообщения","fullName":"api_test.tests.test_delete_message#test_delete_non_exist_message","historyId":"e0db6086c0d7bb6d6ef53031495f4613","time":{"start":1732980799614,"stop":1732980799743,"duration":129},"status":"failed","statusMessage":"Failed: Error: 401","statusTrace":"headers = {'Authorization': 'Bot MTI4NjYwOTgxOTEzNzk5ODg4Mw.Gj2Mlk.YsVgU6PYv7xVW5bl7RYt9DLg2QeCU4OXToIiSw', 'Content-Type': 'application/json'}\ndelete_message_endpoint = <api_test.endpoints.delete_message_endpoint.DeleteMessageEndpoint object at 0x7ff6023d13f0>\n\n    @allure.title(\"Тест на удаление несуществующего сообщения\")\n    def test_delete_non_exist_message(headers, delete_message_endpoint):\n        with allure.step(\"Отправка запроса на удаление сообщения\"):\n            delete_message_endpoint.delete_message(headers, delete_message_endpoint.make_digit(5))\n        with allure.step(\"Проверка статус кода\"):\n>           delete_message_endpoint.check_status_code(404)\n\napi_test/tests/test_delete_message.py:34: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <api_test.endpoints.delete_message_endpoint.DeleteMessageEndpoint object at 0x7ff6023d13f0>\ncode = 404\n\n    def check_status_code(self, code):\n        if self.response.status_code == code:\n            pass\n        else:\n>           pytest.fail(f'Error: {self.response.status_code}')\nE           Failed: Error: 401\n\napi_test/endpoints/base_endpoint.py:35: Failed","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"delete_message_endpoint","time":{"start":1732980799614,"stop":1732980799614,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"headers","time":{"start":1732980799614,"stop":1732980799614,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"testStage":{"status":"failed","statusMessage":"Failed: Error: 401","statusTrace":"headers = {'Authorization': 'Bot MTI4NjYwOTgxOTEzNzk5ODg4Mw.Gj2Mlk.YsVgU6PYv7xVW5bl7RYt9DLg2QeCU4OXToIiSw', 'Content-Type': 'application/json'}\ndelete_message_endpoint = <api_test.endpoints.delete_message_endpoint.DeleteMessageEndpoint object at 0x7ff6023d13f0>\n\n    @allure.title(\"Тест на удаление несуществующего сообщения\")\n    def test_delete_non_exist_message(headers, delete_message_endpoint):\n        with allure.step(\"Отправка запроса на удаление сообщения\"):\n            delete_message_endpoint.delete_message(headers, delete_message_endpoint.make_digit(5))\n        with allure.step(\"Проверка статус кода\"):\n>           delete_message_endpoint.check_status_code(404)\n\napi_test/tests/test_delete_message.py:34: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <api_test.endpoints.delete_message_endpoint.DeleteMessageEndpoint object at 0x7ff6023d13f0>\ncode = 404\n\n    def check_status_code(self, code):\n        if self.response.status_code == code:\n            pass\n        else:\n>           pytest.fail(f'Error: {self.response.status_code}')\nE           Failed: Error: 401\n\napi_test/endpoints/base_endpoint.py:35: Failed","steps":[{"name":"Отправка запроса на удаление сообщения","time":{"start":1732980799614,"stop":1732980799743,"duration":129},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"Проверка статус кода","time":{"start":1732980799743,"stop":1732980799743,"duration":0},"status":"failed","statusMessage":"Failed: Error: 401\n","statusTrace":"  File \"/home/runner/work/diplom/diplom/api_test/tests/test_delete_message.py\", line 34, in test_delete_non_exist_message\n    delete_message_endpoint.check_status_code(404)\n  File \"/home/runner/work/diplom/diplom/api_test/endpoints/base_endpoint.py\", line 35, in check_status_code\n    pytest.fail(f'Error: {self.response.status_code}')\n  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/_pytest/outcomes.py\", line 178, in fail\n    raise Failed(msg=reason, pytrace=pytrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true}],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":2,"attachmentsCount":0,"hasContent":true},"afterStages":[],"labels":[{"name":"parentSuite","value":"api_test.tests"},{"name":"suite","value":"test_delete_message"},{"name":"host","value":"fv-az840-119"},{"name":"thread","value":"2077-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"api_test.tests.test_delete_message"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":2,"broken":0,"skipped":0,"passed":0,"unknown":0,"total":2},"items":[{"uid":"5365b98ab2ad368","reportUrl":"https://prollla.github.io/diplom/1//#testresult/5365b98ab2ad368","status":"failed","statusDetails":"Failed: Error: 401","time":{"start":1732980694678,"stop":1732980694801,"duration":123}}]},"tags":[]},"source":"5b3926c3655b7d5b.json","parameterValues":[]}